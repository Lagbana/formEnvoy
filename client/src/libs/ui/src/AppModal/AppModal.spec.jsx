import React from 'react'
import { shallow, configure } from 'enzyme'
import toJson from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16'
import { AppModal } from '.'

configure({ adapter: new Adapter() })

test('<AppModal /> renders correctly', () => {
  const component = shallow(
    <AppModal
			animationType="slide"
			visible={true}
			presentationStyle="pageSheet"
			onShow={() => null}
			onDismiss={() => null}
			onRequestClose={() => {}}
    />
  )
  expect(toJson(component)).toMatchSnapshot()
})
