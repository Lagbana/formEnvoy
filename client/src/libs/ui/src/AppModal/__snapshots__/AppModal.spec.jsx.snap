// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppModal /> renders correctly 1`] = `
<Component
  animationType="slide"
  hardwareAccelerated={false}
  onDismiss={[Function]}
  onRequestClose={[Function]}
  onShow={[Function]}
  presentationStyle="pageSheet"
  visible={true}
/>
`;
